app/javascript/components/reusable/withUserForm.js

```diff
@@ -417,49 +417,104 @@ const withUserForm = (WrappedComponent, schema, wrappedProps) => {
 
     renderUserRegion(country, state, validateHandler, errors) {
       if (!country || country === 'United States') {
+        const countries = Countries.map(c => c.countryName);
+
         return (
-          <SelectField
-            floatingLabelFixed
-            floatingLabelText={
-              (
-                <FormattedMessage
-                  id='UserForm.state'
-                  defaultMessage='Select State/Region'
-                />
-              )
-            }
-            value={ state }
-            className='userFormInputField state'
-            errorText={ this.errorLanguageHandler('state') }
-            onChange={ this.changeStateHandler }
-            onBlur={ validateHandler('state') }
+          <article className="userFormInputFieldLocationContainer">
+
+            <SelectField
+              style={{ 'min-width': 250 + 'px' } }
```

Comment:
Change made! Originally I had the issue of setting styling for the component since it was an imported material-ui component and my solution at the time was addressing the level of specificity not allowing changes to be made. None the less I found a better solution that I am pushing now for your review!